src/main/java/org/opensearch/ad/transport/IndexAnomalyDetectorRequest.java

```diff
@@ -83,6 +84,22 @@ public IndexAnomalyDetectorRequest(
         this.maxCategoricalFields = maxCategoricalFields;
     }
 
+    public IndexAnomalyDetectorRequest(String detectorID, AnomalyDetector detector, RestRequest.Method method) {
```

Comment:
Where do you call this constructor?

Reply:
from skills repo on creating an anomaly detector from create ad tool, will have PR for that by tomorrow